src/main/java/frc/robot/Auto.java

```diff
@@ -36,45 +45,42 @@ public static BalancePark balancePark(DriveBase db, Gyro pitch, double kp) {
 		return new BalancePark(db, pitch, kp);
 	}
 
+	public static class ActivePark extends CommandBase {
 
+		private final DriveBase drivebase;
+		private final double volts_per_meter;
+		private double linit, rinit;
 
+		public ActivePark(DriveBase db, double p) { // p is the proportional gain, in volts per meter [error]
+			this.drivebase = db;
+			this.volts_per_meter = p;
+		}
 
-    public static class ActivePark extends CommandBase {
-
-        private final DriveBase drivebase;
-        private final double volts_per_meter;
-        private double linit, rinit;
-
-        public ActivePark(DriveBase db, double p) { // p is the proportional gain, in volts per meter [error]
-            this.drivebase = db;
-            this.volts_per_meter = p;
-        }
-
-        @Override
-        public void initialize() {
-            this.linit = this.drivebase.getLeftPosition();
-            this.rinit = this.drivebase.getRightPosition();
-        }
-        @Override
-        public void execute() {
-            double le = this.drivebase.getLeftPosition() - this.linit;
-            double re = this.drivebase.getRightPosition() - this.rinit;
-            this.drivebase.setDriveVoltage(
-                -le * this.volts_per_meter,     // we are assuming that positive position for the encoders is the same direction as positive voltage
-                -re * this.volts_per_meter
-            );
-        }
-        @Override
-        public void end(boolean interrupted) {
-            this.drivebase.setDriveVoltage(0.0, 0.0);
-        }
-        @Override
-        public boolean isFinished() {
-            return false;
-        }
-
-    }
-    public static class BalancePark extends CommandBase {
+		@Override
```

Comment:
This looks like it's just a formatting change (no code change); is that correct?

Reply:
Probably. I did alot of "ctrl+] --> ctrl+[" because there was mixed tabbing and spaces for whitespace.